src/rbdmap

```diff
@@ -1,15 +1,6 @@
 #!/bin/bash
 
 do_map() {
-
-        # default to reasonable value if RBDMAPFILE not set in environment
-        printenv RBDMAPFILE >/dev/null || local RBDMAPFILE=/etc/ceph/rbdmap
-
-	if [ ! -f "$RBDMAPFILE" ]; then
-		logger -p "daemon.warning" -t rbdmap "No $RBDMAPFILE found."
-		exit 0
-	fi
-
```

Comment:
See https://github.com/ceph/ceph/pull/8222 for background. Maybe the test & workunit have outlived their purpose and can be dropped?